### ISSUE TITLE
Dependency on GKSU

### ISSUE THREAD

**Issue by oceanexplorer** [mank319/elementaryPlus#36]
I cloned the repo as documented and ran install.sh and got this error:

```
** (elementaryplus-installer:4566): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-DBEhtDsLUd: Connection refused
sh: 1: gksu: not found
schemas copied

(elementaryplus-installer:4566): GLib-GIO-ERROR **: Settings schema 'apps.elementaryPlusInstaller' is not installed
```
Easily resolved by installing gksu but perhaps it might be considered to remove the dependency or amend the documentation.

**Comment by cybre** [comment]
I might replace gksu with pkexec in the near future, but for now, gksu
stays.

**Comment by mank319** [comment]
@cybre replaced it in his last pull request. I therefore assume, this issue has been fixed ;-)
